### PR TITLE
Styling and markup for the corrections block

### PR DESCRIPTION
**Pull request by helenb**
### What is the context of this PR?
See https://jira.ons.gov.uk/browse/NWP-538
Design: https://www.figma.com/design/Yf4Cpxc8vYdoVrWcJojk2R/New-Website-Platform?node-id=0-1&node-type=canvas&m=dev

### How to review
- Add some notices and corrections to a bulletin / analysis page, and view the result

Note that the wording for the summary element is under discussion as it doesn't work well for plurals at the moment (nunjucks also has no equivalent of django's `pluralize` filter)

### CODE REVIEW

File: ons_alpha/jinja2/templates/components/streamfield/corrections_block.html [hunk 1]
```diff
@@ -1,5 +1,10 @@
...
-{{ value.text|richtext }}
-
-<a href="{{ routablepageurl(page, 'previous_version', version_num) }}">{{ _("View superseded version") }}</a>
+<div class="corrections__item">
+    <div class="corrections__meta">
+        <h2 class="ons-u-fs-r--b corrections__heading">Correction</h2>
+        <time class="corrections__date">{{ value.when|date("DATETIME_FORMAT") }}</time>
+    </div>
+    <div class="corrections__detail">
+        {{ value.text|richtext }}
```
Comment (ababic) [inline ons_alpha/jinja2/templates/components/streamfield/corrections_block.html]: I'm not sure you need to use the `|richtext` filter when working with block values. One to check, I think

File: ons_alpha/jinja2/templates/components/streamfield/corrections_notices_block.html [hunk 2]
```diff
@@ -4,22 +4,44 @@
 {% set notice_blocks = value.blocks_by_name("notice") %}
```
Comment (ababic) [inline ons_alpha/jinja2/templates/components/streamfield/corrections_notices_block.html:4]: Nice digging! I had no idea this method existed.

File: ons_alpha/jinja2/templates/components/streamfield/corrections_notices_block.html [hunk 3]
```diff
@@ -4,22 +4,44 @@
...
+                    </span>
+                {% endif %}
+            </span>
+        </summary>
+        {# {% call onsDetails({"id": "corrections-and-notices", "title": _("Corrections and notices")}) #}
+        {# {% endcall %} #}
+        <div class="ons-container corrections__content ons-u-pt-xl ons-u-pb-3xl">
+            {% if correction_blocks %}
+                {% for block in correction_blocks %}
+                    {% set version_num = loop.index %}
```
Comment (ababic) [inline ons_alpha/jinja2/templates/components/streamfield/corrections_notices_block.html:35]: Bit of a mind-bender, this one... does the index match the actual revision pk, as I think that's what's needed for the link to work?

Comment (ababic) [reply ons_alpha/jinja2/templates/components/streamfield/corrections_notices_block.html:35]: Oh wait, I see that the `previous_version` route handles this. Ignore!

**Review by ababic — APPROVED** [review]

File: ons_alpha/jinja2/templates/components/streamfield/corrections_notices_block.html (hunk 2 above)
Comment (helenb) [reply ons_alpha/jinja2/templates/components/streamfield/corrections_notices_block.html:4]: I wish I could take the credit... I literally only changed markup and styling in this PR, and none of the things you commented one where things I had actually changed :)

File: ons_alpha/jinja2/templates/components/streamfield/corrections_notices_block.html (hunk 3 above)
Comment (helenb) [reply ons_alpha/jinja2/templates/components/streamfield/corrections_notices_block.html:35]: Again, not my code :)

File: ons_alpha/jinja2/templates/components/streamfield/corrections_block.html (hunk 1 above)
Comment (helenb) [reply ons_alpha/jinja2/templates/components/streamfield/corrections_block.html]: I will test this without the filter to see if it makes a difference - it was certainly working using it.

Comment (helenb) [reply ons_alpha/jinja2/templates/components/streamfield/corrections_block.html]: You're right - now removed.

**Pull request merged**